pkg/cli/mirror/operator.go

```diff
@@ -34,6 +34,13 @@ import (
 	"github.com/openshift/oc-mirror/pkg/config"
 	"github.com/openshift/oc-mirror/pkg/image"
 	"github.com/openshift/oc-mirror/pkg/operator"
+	"k8s.io/klog/v2"
+)
+
+const (
```

Comment:
A possible rebasing issue? These were removed from this file on a previous merge,